proto/status/v1/status.proto

```
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package status.v1;
```

Buf (1.54.0)

Files with package "status.v1" must be within a directory "status/v1" relative to root but were in directory "proto/status/v1".

(PACKAGE_DIRECTORY_MATCH)